### PR TITLE
Add a converter from an annotated string field value to an array of

### PR DESCRIPTION
**Pull request by toregge**
strings representing the indexing terms.

Multiple indexing terms for same position are placed in a nested array.

@geirst : please review

### CODE REVIEW

File: searchsummary/src/vespa/searchsummary/docsummary/linguistics_tokens_converter.h [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+namespace document {
+
+class FieldValue;
+class Span;
+
+}
+
+namespace search::docsummary {
+
+class IJuniperConverter;
```
Comment (geirst) [inline searchsummary/src/vespa/searchsummary/docsummary/linguistics_tokens_converter.h:16]: Is this needed? If not, please remove.

File: searchsummary/src/vespa/searchsummary/docsummary/linguistics_tokens_converter.h [hunk 2]
```diff
@@ -0,0 +1,37 @@
+// Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include "i_string_field_converter.h"
+
+namespace document {
+
+class FieldValue;
```
Comment (geirst) [inline searchsummary/src/vespa/searchsummary/docsummary/linguistics_tokens_converter.h:9]: Are these needed? If not, please remove.

**Review by geirst — DISMISSED** [review]
👍

File: searchsummary/src/vespa/searchsummary/docsummary/linguistics_tokens_converter.h (hunk 1 above)
Comment (toregge) [reply searchsummary/src/vespa/searchsummary/docsummary/linguistics_tokens_converter.h]: Removed.

File: searchsummary/src/vespa/searchsummary/docsummary/linguistics_tokens_converter.h (hunk 2 above)
Comment (toregge) [reply searchsummary/src/vespa/searchsummary/docsummary/linguistics_tokens_converter.h]: Removed.

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**